src/phoenix/experimental/evals/models/base.py

```diff
@@ -123,6 +148,11 @@ async def agenerate(self, prompts: List[str], instruction: Optional[str] = None)
             raise e
         return result
 
+    def _verbose_generation_info(self) -> str:
+        # if defined, returns additional model-specific information to display if `generate` is
+        # run with `verbose=True`
+        return ""
```

Comment:
Suggestion: I think we can do without this for now. The more lightweight we keep the base class, the better.

Reply:
I'd argue that this is a pretty inobtrusive way to implement model-implementation-specific printing of verbose messages for concrete classes. The alternative is to always pass a `verbose` flag downstream and ask that each specific model class handle this argument.

`_verbose_generation_info` is not required to be implemented and serves as a hook to print additional model-specific information. Putting it on the base class allows us to implement this `verbose-mode` printing in a uniform way across all classes.

Reply:
finished discussion offline